assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +71,99 @@ angular.module('openshiftConsole')
       console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
     };
 
+    var podRelationships = function() {
+      $scope.servicePodsNoDeployment = {};
+      $scope.solitaryPods = {};
+
+      angular.forEach($scope.pods, function(pod, name){
+        var replicationcontrollers = [];
+        var services = [];
+        angular.forEach($scope.deployments, function(deployment, depName){
+          var ls = new LabelSelector(deployment.spec.selector);
+          if (ls.matches(pod)) {
+            replicationcontrollers.push(depName);
+          }
+        });
+        angular.forEach($scope.services, function(service, svcName){
+          var ls = new LabelSelector(service.spec.selector);
+          if (ls.matches(pod)) {
+            services.push(svcName);
+            if (replicationcontrollers.length == 0) {
+              $scope.servicePodsNoDeployment[svcName] = $scope.servicePodsNoDeployment[svcName] || {};
+              $scope.servicePodsNoDeployment[svcName][name] = pod;
+            }
+          }
+        });
+        if (replicationcontrollers.length == 0 && services.length == 0) {
+          $scope.solitaryPods[name] = pod;
+        }
+      });
+    };
+
+    var deploymentPods = function() {
+      $scope.deploymentPods = {};
+      angular.forEach($scope.deployments, function(deployment, name) {
+        var ls = new LabelSelector(deployment.spec.selector);
+        $scope.deploymentPods[name] = ls.select($scope.pods);
+      });
+
+      console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
+    };    
+
+    var serviceDeploymentConfigs = function() {
+      $scope.serviceDeploymentConfigs = {};
+      angular.forEach($scope.deploymentConfigs, function(deploymentConfig, depName){
+        if (!deploymentConfig) {
```

Comment:
if the view is no longer looping over $scope.deploymentConfigs, I don't think we need this